apps/client/src/context.js

```diff
@@ -1,3 +1,33 @@
-import React from "react";
-export const defaultValues = { topic: null, address: null, checker: null };
-export default React.createContext();
+import React, { useReducer, useEffect, createContext } from "react";
+const session = JSON.parse(sessionStorage.getItem("vergunningCheckData"));
+
+let reducer = (data, newData) => {
+  if (newData === null) {
+    sessionStorage.removeItem("vergunningCheckData");
+    return defaultValues;
+  }
+  return { ...data, ...newData };
+};
+
+const defaultValues = session
+  ? session
+  : { topic: null, address: null, questionId: 0 };
```

Comment:
Why is this called `questionId` and not `questionIndex`? The real `questionId` is a hash and not a number